docker/Dockerfile

```diff
@@ -20,20 +20,20 @@
 FROM centos:7
 MAINTAINER Apache BookKeeper <dev@bookkeeper.apache.org>
 
-ARG BK_VERSION=4.9.0
+ARG BK_VERSION=4.11.0
```

Comment:
I am not sure that the bash files on 4.9.2 work with java11 due to removed JVM command line flags.

4.10 should work.

Btw we are close to releasing 4.12 so 4.9 is becoming far distant from current  versions

Reply:
Also It would be better to use the version in pom files and in current master it should be 4.12.0-SNAPSHOT but if I change it the image cannot be built so currently I can only use this version.
We are in the middle of the process of releasing 4 11.1 so probably we will set 4.11.1 here on current master.

Another solution would be to pick the binaries from local filesystem if the version is a SNAPSHOT

Reply:
This is my fix for using JRE
that said, it won't be available to old BK versions
https://github.com/apache/bookkeeper/pull/2436

with BK 4.12.0 we will be able to use JRE 11